src/pages/detail/tabs/intro/GroupIntroTab.tsx

```
@@ -6,28 +6,32 @@ import "prismjs/themes/prism-tomorrow.css";
 import "./styles.css";
 
-import useSWR from "swr";
+import { getGroup } from "@/apis/group";
 import { getNotionPage } from "@/apis/notion";
 import { useParams } from "react-router-dom";
-import { getGroup } from "@/apis/group";
+import useSWR from "swr";
+import NotionWrapper from "./NotionWrapper";
 //import NotionWrapper from "./NotionWrapper";
 
 interface GroupIntroTabProps {}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)